.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "files.eol": "\n"
+}
```

Comment:
Well that's interesting. Did not know this was something that we'd want to do. No biggie.